### DISCUSSION TITLE
"Fallback to default locale" checkbox in block field

### DISCUSSION THREAD

**Discussion by unonweb** [payloadcms/payload#1868] [Q&A]
Hi there,

this checkbox shown below appeared suddenly, I don't know why.

![Screenshot at 2023-01-13 11-07-25](https://user-images.githubusercontent.com/79059627/212304919-3e21a362-9fd4-416f-9e3a-23160d031b28.png)

My payload.config.js:
```js
localization: {
	locales: [
		'de',
		'en',
	],
	defaultLocale: 'de',
	fallback: true, 
}
```
But the checkbox doesn't react to `fallback: false` anyway.
```js
{
	type: 'blocks',
	name: 'blocksMain',
	label: 'Main Layout',
	localized: false,
	blocks: [
		layoutFlexColumns
	]
}
```
No change if I set `localized: true`

Apparently it only shows up in **block** field type.

**Comment by JessRynkar** [comment]
Hi @unonweb - the 'Fallback to default locale' checkbox was added in a recent release, you can see the PR and issue that brought about this change here: https://github.com/payloadcms/payload/pull/1719

The checkbox will only appear when there are no items your array, so if you click 'Add main layout' you should see it disappear and this is expected behavior.

However, if you have `localized: false` it would make sense to me that this checkbox is hidden - I will take this to the team and get back to you.

**Comment by DanRibbens** [comment]
I was able to recreate this problem and I've made a fix PR for it. https://github.com/payloadcms/payload/pull/2316

This will be available on the next release.
